deploy/eks/action.yml

```diff
@@ -17,11 +17,34 @@ inputs:
   AWS_REGION:
     description: AWS region to deploy to stored as secret
     required: true
-
+  SLACK_SUCCESS_CHANNEL_ID:
+    description: The Slack channel ID(s) to send successful deployment notifications
+    required: false
+  SLACK_FAILURE_CHANNEL_ID:
```

Comment:
idea: can we default to the success on failure channel in case, its the only one provided. Let's say a team uses on channel only for ALL their CI runs

Reply:
Yes, I like this!

Reply:
Ok, I think I remember why I didn't do this:

**Scenario 1:** 
No Slack messages desired
`FAIL` = n/a
`SUCCESS` = n/a

**Scenario 2:**
Failure messages only
`FAIL` = `fail_channel`
`SUCCESS` = n/a

**Scenario 3:**
Failure and success messages to the same channel:
`FAIL` = `single_channel`
`SUCCESS` = `single_channel`

**Scenario 4:**
Failure and success to different channels
`FAIL` = `fail_channel`
`SUCCESS` = `success_channel`

So the only default scheme I could think of is having `FAIL` default to `SUCCESS` if `FAIL` isn't provided. This makes naming kind of tricky, and I find it makes the interface a bit muddier vs. just passing the same channel twice. I'm also 90% certain you can't dynamically access another `input` in the `default` of another, so this would add another step to maintain/test as well.

Thoughts? To me the repeated config is just less confusing. I do wonder if absolutely no one will use different channels and we can skip this all together...

Reply:
That totally makes sense! I think we're more likely to post on failure, so the current implementation works the best for this.